include/highfive/bits/H5Annotate_traits.hpp

```diff
@@ -31,19 +31,31 @@ class AnnotateTraits {
                               const DataSpace& space, const DataType& type);
 
     ///
-    /// \brief createDataSet create a new dataset in the current file with a
+    /// \brief createAttribute create a new attribute on the current dataset with
     /// size specified by space
-    /// \param dataset_name identifier of the dataset
+    /// \param dataset_name identifier of the attribute
     /// \param space Associated DataSpace, see \ref DataSpace for more
     /// informations
-    /// \return DataSet Object
+    /// \return Attribute Object
     ///
     ///
     ///
     template <typename Type>
     Attribute createAttribute(const std::string& attribute_name,
                               const DataSpace& space);
 
+    ///
+    /// \brief writeAttribute create a new attribute on the current dataset and
+    /// writes to it; must support DataSpace::From
+    /// \param attribute_name identifier of the attribute
+    /// \param data Associated data to write, see \ref DataSpace for more
+    /// information on DataSpace::From
+    /// \return Attribute Object
+    ///
+    template <typename Type, typename InferredType>
+    Attribute writeAttribute(const std::string& attribute_name,
```

Comment:
I'd prefer having an overload of createAttribute. The overloading can't be ambiguous because all DataSpace constructors are explicit and having just one name makes it easier to remember.

Reply:
Since both return the same thing, and because of the unambiguity, I think that's fine. I'll change it.

Reply:
Changed.